### PR TITLE
Issue #2: Add Gradle Task to generate Usage Documentation

### PR DESCRIPTION
**Pull request by averabaq**
Issue #2: Add Gradle Task to generate Usage Documentation

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -37,6 +42,23 @@ dependencies {
 }
 
 tasks {
+    val fatJar = getByName<ShadowJar>("shadowJar") {
+        manifest {
+            attributes["Main-Class"] = "com.cjbooms.fabrikt.cli.CodeGen"
+            attributes["Implementation-Title"] = "fabrikt"
+            attributes["Implementation-Version"] = project.version
```
Comment (averabaq) [inline build.gradle.kts:49]: Not sure how you want to manage the versioning @cjbooms

Comment (cjbooms) [reply build.gradle.kts:49]: We should go semver for the public release I guess

Comment (cjbooms) [reply build.gradle.kts:49]: I guess we can start at 1.0 saying as this is a port of a successful internal project

**Comment by cjbooms** [comment]
As part of this can we update the README with the usage docs?

**Pull request merged**